### PR TITLE
PR to master with all commits

### PR DESCRIPTION
**Pull request by kirylpl**

### CODE REVIEW

**Comment by kirylpl** [comment]
@anatoliliotych third PR created

**Comment by kirylpl** [comment]
@anatoliliotych done

File: .env.test [hunk 1]
```diff
@@ -0,0 +1 @@
+TG_TOKEN=427789043:AAGEawYOrOJ6ZblBzU-ObDJxjexyPhQGMXw
```
Comment (anatoliliotych) [inline .env.test]: I hope this is test token

File: bot.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+require 'nokogiri'
+require 'telegram/bot'
+require 'open-uri'
+require 'logger'
+
+TOKEN = ENV['TG_TOKEN'].freeze
+URL = 'http://bash.im/best'.freeze
+$posts = []
+
+def initialize
```
Comment (anatoliliotych) [inline bot.rb]: this method looks strange, when I say that we need to move logger to initializer, I meant that we need to path logger object to bot initializer here: Telegram::Bot::Client.run(TOKEN)

File: bot.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          chat_id: message.chat.id,
+          text: $posts.first(10).join("\n \n").to_s
+        )
+      when '/refresh'
+        parse_posts
+      # todo: when '/stop'
+      when '/help'
+        bot.api.send_message(
+          chat_id: message.chat.id,
+          text: "Avaialable commands: \n
```
Comment (anatoliliotych) [inline bot.rb]: use single quotes

File: .env.test (hunk 1 above)
Comment (kirylpl) [inline .env.test]: Yes, it differs from main token

**Comment by kirylpl** [comment]
@anatoliliotych new commit arrived, please check

File: bot.rb [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        )
+      else
+        bot.api.send_message(
+          chat_id: message.chat.id,
+          text: 'Sorry, IDK what you want from me'
+        )
+      end
+    end
+  rescue Telegram::Bot::Exceptions::ResponseError => err
+  logger = Logger.new('logfile.log')
```
Comment (anatoliliotych) [inline bot.rb]: check style and we don't log error, we log traceback and error message

**Comment by kirylpl** [comment]
@anatoliliotych

**Pull request merged**